util/ranger/points.go

```diff
@@ -250,6 +250,28 @@ func (r *builder) buildFormBinOp(expr *expression.ScalarFunction) []point {
 		return nil
 	}
 
+	// if the colunm is unsigned integer and the value
+	// is less than zero, we only need compare with zero.
+	if mysql.HasUnsignedFlag(ft.Flag) {
```

Comment:
Can we extract the part as a function like `handleUnsignedIntCol`?

Reply:
ok